src/Main/Main.js

```diff
@@ -0,0 +1,7 @@
+import * as CalendarUtils from '../utils/CalendarUtils/CalendarUtils';
+import BasicCalendar from '../resources/BasicCalendar/BasicCalendar.json';
+
+// eslint-disable-next-line no-unused-vars
+export const processParams = (container, gitHubUsers, gitLabUsers, proxyUrl) => {
```

Comment:
Why are you declared unused variables?

Reply:
These'll be the expected variables later on.

Put simply, we'll use them. :)

Reply:
Wouldn't better if not to look ahead in the code? In this case, probably we'll use them in the next PRs, but if this not probably, it doesn't too pretty.

Otherwise, the code looks good so this is just a little remark. :D

Reply:
Agreed, let's develop incrementally. If this is not needed for now let's remove it.